### PR TITLE
Admission Controllers 101 Docs

### LINKED ISSUE #857: [Enhancement] Admission Controllers 101
### Description

It would be nice to have a section in the docs which is a better, easier, more comprehensive introduction to admission controllers with some nice, perhaps animated, graphics so users get more comfortable with what they're installing and how generally they work.

### Slack discussion

_No response_

### PR DESCRIPTION
**Pull request by 1Shubham7**
## Related issue #

Admission Controllers is a part of Introduction section of the Docs. Closes #857 

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

File: content/en/docs/Introduction/admission-controllers.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+For example, the LimitRanger admission controller can augment pods with default resource requests and limits (mutating phase) if the request doesn't specify it already, as well as it can verify that the pods with explicitly set resource requirements do not exceed the per-namespace limits specified in the LimitRange object (validating phase). Thus acting as a combination of both Mutating as well as Validating admission controller.
+
+## Why do we need admission controllers?
+
+Admission controllers were introduced in Kubernetes as a set of Plugins for improving the cluster security. Admission controllers must be enabled to use some of the more advanced security features of Kubernetes such as Pod security Policies.
+
+Some aspects of the built-in Kubernetes’ operations are in fact governed by admission controllers. For example, when a namespace is deleted and subsequently enters the Terminating state, the NamespaceLifecycle admission controller is what prevents any new objects from being created in this namespace.
```
Comment (anushkamittal2001) [inline content/en/docs/Introduction/admission-controllers.md:25]: "Some aspects of the built-in Kubernetes operations are in fact governed by admission controllers." IMO an apostrophe is not required

**Review by anushkamittal2001 — APPROVED** [review]
A nitpick really, lgtm

File: content/en/docs/Introduction/admission-controllers.md (hunk 1 above)
Comment (1Shubham7) [reply content/en/docs/Introduction/admission-controllers.md:25]: yup, removing it.

File: content/en/docs/Introduction/admission-controllers.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+kube-apiserver -h | grep enable-admission-plugins
+```
+
+disable-admission-plugins flag is used to disable a comma-delimited list of admission controllers. This list may also include admission controllers enabled by default.
+
+```sh
+kube-apiserver --disable-admission-plugins=PodNodeSelector,AlwaysDeny
+```
+
+## Admission Controllers (Kubernetes v1.28)
```
Comment (chipzoller) [inline content/en/docs/Introduction/admission-controllers.md:63]: This entire section is unnecessary for a 101-level introduction to admission controllers. It's fine to list a couple and explain what they are and what they do and then provide a link to the docs, but this document isn't about being a reference guide.

File: content/en/docs/Introduction/admission-controllers.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
+---
+title: Admission Controllers
+description: A Guide to Kubernetes Admission Controllers
+weight: 35
+---
+
+## What are Admission Controllers?
```
[the remark this answers is not in the document]
Comment (chipzoller) [reply content/en/docs/Introduction/admission-controllers.md:7]: This section should also introduce the concept of "built-in" admission controllers and external, or dynamic, admission controllers which is what Kyverno is.

File: content/en/docs/Introduction/admission-controllers.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+<img src="assets/kubernetes-admission-controllers.png" alt="Kubernetes Admission Controllers" />
+
+## Admission Controllers Phases
+
+Admission Controllers can be thought of as a gatekeeper to the API server. They intercept incoming API requests and may change the request object or deny the request altogether. This process of admission control has two phases: the mutating phase (where mutating admission controllers are run), followed by the validating phase (where validating admission controllers are run). Some of the admission controllers act as a combination of both. If any of the admission controllers reject the request, the entire request is rejected and an error is returned to the end-user.
+
+For example, the LimitRanger admission controller can augment pods with default resource requests and limits (mutating phase) if the request doesn't specify it already, as well as it can verify that the pods with explicitly set resource requirements do not exceed the per-namespace limits specified in the LimitRange object (validating phase). Thus acting as a combination of both Mutating as well as Validating admission controller.
+
+## Why do we need admission controllers?
```
Comment (chipzoller) [inline content/en/docs/Introduction/admission-controllers.md:21]: I would put this section before the phases.

File: content/en/docs/Introduction/admission-controllers.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+```sh
+kube-apiserver -h | grep enable-admission-plugins
+```
+
+disable-admission-plugins flag is used to disable a comma-delimited list of admission controllers. This list may also include admission controllers enabled by default.
+
+```sh
+kube-apiserver --disable-admission-plugins=PodNodeSelector,AlwaysDeny
+```
```
Comment (chipzoller) [inline content/en/docs/Introduction/admission-controllers.md:61]: I would make this section about webhook-based admission controllers and less about enabling built-in admission controllers. It's nice to add a small paragraph on how to work with the built-in ones, but this is Kyverno and so the focus should be on understanding external admission controllers.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Thanks for attempting this document. It's a lot of work and I've tried to add some helpful comments without being too specific. In general, these are the following ideas we want to capture:

* What is the _concept_ of controlling admission
* Why is this useful
* What some of the options available (built-in and external (dynamic)
* What are the phases
* (Dive into external at this point) Understanding webhooks

**Pull request closed without merging**